### ISSUE TITLE
Getting Vermy to spawn more

### ISSUE THREAD

**Issue by NotBlaze** [BLCM/BLCMods#293]
Hey guys, sorry to bother  again! Personally I want vermy to spawn just a little bit faster, and I've been going through the community patch v3 and am having troubles making him spawn more. Could somebody help me with the code? All i wanna do is make him spawn more.

**Comment by Greeem** [comment]
How much more? I could probably make it like a LOT if you want me to.

**Comment by VariantLoki** [comment]
I took a look at the patch and it looks like these are the lines you will need to change to increase his spawnrate

GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase1,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.700000,.85",",GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase2,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.400000,.55",",GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase3,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.200000,.35",",GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase4,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.100000,.25",",GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase5,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.075000,.20"

Most likely if you change the Phase 5 value, that will increase the chance that Ultimate Badass Varkids will evolve into vermi.

**Comment by Rhysand** [comment]
I have been trying to get vermi to spawn 100% of the time, so every varkid must go through every phase. I tried screwing with the numbers of the phases for quite a while and nothing seemed to be happening. I noticed that in shadow's patch 2.0 he increased the spawn rate of vermi by 15%, and in 3.0 he further increased the chance to 20%. I still have the 2.0 code, and I compared them. The only change in the code was the .20 at the end of the final phase 5 was a .15 in patch 2.0. So I tried changing only that number to .99. Still no dice. 
Community Patch 2.0: GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase5,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.075000,.15"
Community Patch 3.0: GD_Balance.WeightingPlayerCount.BugmorphCocoon_PerPlayers_Phase5,ConditionalInitialization.ConditionalExpressionList[4].BaseValueIfTrue.BaseValueConstant,0.075000,.20"
Maybe it has something to do with the amount of players? I tried all of these changes single player.
